Comparisons

louisgosschalk opened the pull request:
Dit is ongeveer hoe ik van plan ben om comparisons op te bouwen

graag commentaar

louisgosschalk committed:
simple setup

louisgosschalk committed:
opbouw comparison idee

JensLangerak commented:
Het klinkt allemaal logisch. die event class, gaat dat een omschrijving van een event worden die vervolgens in de data wordt gezocht? Of gaat dat een verzameling rijen bevatten die samen een event vormen? of nog weer wat anders?

louisgosschalk committed:
Event higher level

Events are going to be used in multiple C’s, so it’s not the same level
as comparison
